app/models/User.scala

```diff
@@ -108,7 +110,28 @@ case class MiniUser(
    id: UUID,
    fullName: String,
    avatarURL: String,
-   email: Option[String])
+   email: Option[String]) {
+      /**
+       * return MiniUser as string in jsonld format, w/fullName split into first and last
+       */
+       def to_jsonld () : String = {
+          var firstName= "";
+          var lastName= "";
+          if(fullName.split("\\w+").length>1){
+             lastName = fullName.substring(fullName.lastIndexOf(" ")+1);
+             firstName = fullName.substring(0, fullName.lastIndexOf(' '));
+           } else{ firstName = fullName; }
+	   val authorLD = JsObject(Seq(
+                   "@type" -> JsString("Person"),
+                   "givenName" -> JsString(firstName.toString()),
+                   "familyName" -> JsString(lastName.toString()),
+                   "email" ->  JsString(email.get.toString()),
```

Comment:
careful, email.get might return None for example for twitter, and throw an exception.

Reply:
your next comment helped